personal_assistant/main.py

```diff
@@ -44,80 +47,76 @@ def main():
             print(greeting(), end=" ")
 
         elif command == "add":
-            print(add_contact(personal_assistant, args))
+            print(add_contact(args))
 
         elif command == "all":
-            [print(c) for c in show_all(personal_assistant)]
+            print(show_all())
 
         elif command == "phone":
-            print(show_phone(personal_assistant, args))
+            print(show_phone(args))
 
         elif command == "change":
-            print(change_contact(personal_assistant, args))
+            print(change_contact(args))
 
         elif command == "remove-phone":
-            print(remove_number(personal_assistant, args))
+            print(remove_number(args))
 
         elif command == "add-birthday":
-            print(add_bd(personal_assistant, args))
+            print(add_bd(args))
 
         elif command == "show-birthday":
-            print(show_birthday(personal_assistant, args))
+            print(show_birthday(args))
 
         elif command == "birthdays":
-            [print(bd) for bd in birthdays_next_week(personal_assistant)]
+            [print(bd) for bd in birthdays_next_week()]
 
         elif command in ["delete", "remove"]:
-            print(del_contact(personal_assistant, args))
+            print(del_contact(args))
 
         elif command == "entries":
-            print(num_records(personal_assistant))
+            print(num_records())
 
         elif command == "help":
             print(messages.contacts_menu + messages.notes_menu)
 
         elif command == "add-note":
-            print(personal_assistant.add_note())
+            print(add_note())
+
+        elif command == "edit-note":
+            print(edit_note())
 
         elif command == "remove-note":
-            note_name = input("Enter the name of the note to delete: ")
-            print(personal_assistant.remove_note(note_name))
+            print(remove_note())
 
         elif command == "show-notes":
-            print(personal_assistant.get_all_notes())
-
-        elif command == "edit-note":
-            note_name = input("Enter the name of the note you want to edit: ")
-            new_text = input("Enter the new text for the note: ")
-            print(personal_assistant.edit_note(note_name, new_text))
+            print(show_notes())
 
         elif command == "search-note":
```

Comment:
we need to specify on the documentation or on the prompt that we need to enter the search params like "tag=<text>". Otherwise typical use cases will be like this:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/5471b445-a000-474f-a63e-532134a019ce)
@DmytroVasylenko6 also FYI - please resolve it after you read it